pkg/cfn/manager/api.go

```diff
@@ -92,6 +92,8 @@ func (c *StackCollection) doCreateStackRequest(i *Stack, templateBody []byte, ta
 // any errors will be written to errs channel, when nil is written,
 // assume completion, do not expect more then one error value on the
 // channel, it's closed immediately after it is written to
+//
+// StackCollection doesn't dedup the tags passed as an argument and the StackCollection's shared tags.
```

Comment:
I don't think this is really important, let's keep documentation here as is, otherwise there is much more that can be documented about this or any other function.

Reply:
Understood! Fixed in the latest commit.